### PR TITLE
Remove deprecated symbols sheduled for/after 2.089

### PR DESCRIPTION
**Pull request by MoonlightSentinel**

### CODE REVIEW

**Comment by MoonlightSentinel** [comment]
@thewilsonator Removed the `enforceEx` change as it requires `vibe-d` changes and added changelogs

**Comment by thewilsonator** [comment]
Auto-merge toggled on

File: changelog/binaryReverseArgs.dd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Deprecated `std.functional.binaryReverseArgs was removed
```
Comment (Geod24) [inline changelog/binaryReverseArgs.dd]: Missing backquote

Comment (MoonlightSentinel) [reply changelog/binaryReverseArgs.dd]: Whoops => #7291 

Is it possible to generate a preview of the entire changelog?

Comment (wilzbach) [reply changelog/binaryReverseArgs.dd]: It is also automatically generated on DAutoTest, e.g. http://dtest.dlang.io/artifact/website-08a5fe1c6137cc5879028d65e8b1ee42781468bc-a53a4a2d720fb1bbe0aa8dcd509e9797/web/changelog/pending.html for this PR

**Pull request merged**